### ISSUE TITLE
Question about torch.svd

### ISSUE THREAD

**Issue by XMU-qcj** [paul007pl/GMCNet#5]
svd_cpu: the updating process of SBDSDC did not converge (error: 2)

**Comment by paul007pl** [comment]
I also met this issue frequently for clean data registration.
I think it should be a bug with the svd function, because many previous methods also observe this issue.

Based on my experience, here are two suggestions:
1. Use another random seed, and re-train.
2. Using the hyper-parameter estimation sub-network may help.
